flask_testing/utils.py

```diff
@@ -381,15 +378,23 @@ def _spawn_live_server(self):
 
         self._process.start()
 
-        # we must wait for the server to start listening with a maximum timeout of 5 seconds
-        timeout = 5
-        while timeout > 0:
-            time.sleep(1)
+        # We must wait for the server to start listening, with a maximum
+        # timeout of the amount of seconds specified in the LIVESERVER_TIMEOUT
+        # setting. Defaults to 5 seconds.
+        timeout = self.app.config.get('LIVESERVER_TIMEOUT', 5)
```

Comment:
To be consistent with the config key names of other extensions, let's prefix this with `TESTING_`